os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -415,6 +422,20 @@ int8_t cmd_wifi_ap(trwifi_softap_config_s *softap_config)
 	return ret;
 }
 
+void wlan_store_connect_info(void)
+{
+	u8 *ifname[1] = {(u8*)WLAN0_NAME};
+	rtw_wifi_setting_t setting;
+
+	if(wifi_get_setting((const char*)ifname[0],&setting) || setting.mode == RTW_MODE_AP){
+		RTW_API_INFO("\r\n %s():wifi_get_setting fail or ap mode", __func__);
+		return;
+	}
+
+	rtw_memcpy(&wifi_ap_info.ssid, setting.ssid, strlen((char const*)setting.ssid));

```

Comment:
strlen doesn't contain null character. So it can cause side effect. So you should copy strlen(ssid) + 1

Reply:
> strlen doesn't contain null character. So it can cause side effect. So you should copy strlen(ssid) + 1

Done.